miio/fan_miot.py

```diff
@@ -525,3 +548,260 @@ def delay_off(self, minutes: int):
             raise FanException("Invalid value for a delayed turn off: %s" % minutes)
 
         return self.set_property("power_off_time", minutes)
+
+
+class OperationModeFanZA5(enum.Enum):
+    Nature = 0
+    Normal = 1
+
+
+class FanStatusZA5(DeviceStatus):
+    """Container for status reports for FanZA5."""
+
+    def __init__(self, data: Dict[str, Any]) -> None:
+        self.data = data
```

Comment:
Please add an example input for the data.

Reply:
Example added. Is the format OK?

Reply:
Looks good to me, you can use that as an input to the unit tests of the status class, too!